lib/src/single_line_comments/single_line_comment.dart

```diff
@@ -14,11 +14,15 @@ class SingleLineComment {
   /// The comment text with the characters signifying the comment.
   final String outerContent;
 
+  //source, which provided comment to parse
```

Comment:
1. Doc comments should start with `///`. It's only then that they are compiled into the package documentation and are shown in IDE tips.

2. Every example in the guide has a single space after `//` or `///`, starts with a capital letter, and ends with a period: https://dart.dev/guides/language/effective-dart/documentation

3. In English, there should be no comma here.

4. According to the guide, we should PREFER using this when referring to the current object, like "this comment": https://dart.dev/guides/language/effective-dart/documentation#prefer-using-this-instead-of-the-to-refer-to-a-members-instance

5. "Source" is an unclear noun that leaves questions. I would have put it like this:

```dart
/// The object this comment was parsed from, if any.
///
/// The object class is specific to the parser.
```

Reply:
Thank you